### DISCUSSION TITLE
Default Working Directory

### DISCUSSION THREAD

**Discussion by rzelnik** [beakerbrowser/beaker#1552] [General] [answered]
In the older versions of Beaker there was a "Default working directory" setting, that allowed to access (and edit) Dat files outside the browser. Is there any way how to do that in the new Beaker?

**Comment by pfrazee** [comment, marked as answer]
Unfortunately this got dropped to cut down on some complexity (both user-facing and internal). The `hyperdrive-daemon` CLI has two useful tools for this: import/export and a FUSE interface. You can use that tool with Beaker.

We're waiting to see whether FUSE can potentially replace the old folder-sync.

**Reply by lumenpink** [reply]
FUSE in Hyperdrive client is an essential tool. Hope it could run in windows soon too.
